.yo-rc.json

```diff
@@ -1,10 +1,126 @@
 {
+    "generator-jhipster": {
```

Comment:
Should this be indented 2 spaces instead of 4?

Reply:
prettier config.